koin-projects/examples/multimodule-ktor/app/src/main/kotlin/org/koin/samples/multi_module/app/application.kt

```diff
@@ -0,0 +1,55 @@
+package org.koin.samples.multi_module.app
+
+import io.ktor.application.Application
+import io.ktor.application.call
+import io.ktor.application.install
+import io.ktor.features.CallLogging
+import io.ktor.response.respondText
+import io.ktor.routing.get
+import io.ktor.routing.routing
+import io.ktor.server.engine.commandLineEnvironment
+import io.ktor.server.engine.embeddedServer
+import io.ktor.server.netty.Netty
+import org.koin.ktor.ext.getKoin
+import org.koin.ktor.ext.inject
+import org.koin.ktor.ext.koin
+import org.koin.logger.slf4jLogger
+import org.koin.samples.multi_module.common.IService
+import org.koin.samples.multi_module.module_a.ModuleAService
+import org.koin.samples.multi_module.module_a.moduleA
+import org.koin.samples.multi_module.module_b.ModuleBService
+import org.koin.samples.multi_module.module_b.moduleB
+
+fun Application.main() {
+    install(CallLogging)
+    koin {
```

Comment:
I would keep `Install(Koin)` as it's a Ktor feature

Reply:
But this is an example in the style of Ktor DSL in the Ktor module. for many modules they have already switched to it. They even have an example on the home page in the DSL style: https://github.com/ktorio/ktor